proto/cosmos/quarantine/v1beta1/quarantine.proto

```diff
@@ -0,0 +1,71 @@
+syntax = "proto3";
+package cosmos.quarantine.v1beta1;
+
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos_proto/cosmos.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/quarantine";
+
+// QuarantinedFunds defines structure that represents coins that have been quarantined.
+message QuarantinedFunds {
+  // to_address is the intended recipient of the coins that have been quarantined.
+  string to_address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // unaccepted_from_addresses are the senders that have not been part of an accept yet for these coins.
+  repeated string unaccepted_from_addresses = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // coins is the amount currently in quarantined for the two addresses.
+  repeated cosmos.base.v1beta1.Coin coins = 3
+      [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"];
+  // declined is true if these funds were previously declined.
+  bool declined = 4;
+}
+
+// AutoResponseEntry defines the auto response to one address from another.
+message AutoResponseEntry {
+  // to_address is the receiving address.
+  string to_address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // from_address is the sending address.
+  string from_address = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // response is the auto-response setting for these two addresses.
+  AutoResponse response = 3;
+}
+
+// AutoResponseUpdate defines a quarantine auto response update that should be applied.
+message AutoResponseUpdate {
+  // from_address is the address that funds would be coming from.
+  string from_address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+
+  // response is the automatic action to take on funds sent from from_address, or in the
+  // case of AUTO_RESPONSE_UNSPECIFIED, that no automatic action should be taken.
+  AutoResponse response = 2;
+}
+
+// AutoResponse enumerates the quarantine auto-response options.
+enum AutoResponse {
+  option (gogoproto.goproto_enum_prefix) = false;
+
+  // AUTO_RESPONSE_UNSPECIFIED defines that no automatic action should be taken.
+  AUTO_RESPONSE_UNSPECIFIED = 0;
```

Comment:
Use `off` and remove references to `unspecified`

```suggestion
  AUTO_RESPONSE_OFF = 0;
```

Reply:
Guidelines for protos say that the zero value should be "Unspecified".

Reply:
Yes. I forgot that about enums.